apps/back-office/src/app/melding/[meldingId]/_components/BackLink.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react'
+
+import { BackLink } from './BackLink'
+
+describe('BackLink', () => {
+  it('renders an extra class name', () => {
+    render(<BackLink className="extra" href="#" />)
+
+    const component = screen.getByRole('link')
+
+    expect(component).toHaveClass('ams-link ams-link--standalone _link_9d6d91 extra')
```

Comment:
I don't think the CSS modules hash is idempotent, so this will probably easily lead to failed tests.

To check if we correctly pass the classname, we probably only need this:

```suggestion
    expect(component).toHaveClass('ams-link extra')
```